discovery-provider/alembic/versions/2fad3671bf9f_backfill_artist_pick.py

```diff
@@ -40,10 +40,17 @@ def build_sql(up, env):
         'handles': handles,
         'pinned_track_ids': pinned_track_ids,
     }
+    # only backfill records last updated before the csvs were pulled
     if up:
-        inner_sql = "UPDATE users SET artist_pick_track_id = data_table.pinned_track_id FROM (SELECT unnest(:handles) AS handle, unnest(:pinned_track_ids) AS pinned_track_id) AS data_table WHERE users.is_current = True AND users.handle = data_table.handle;"
+        inner_sql = """UPDATE users
+        SET artist_pick_track_id = data_table.pinned_track_id
+        FROM (SELECT unnest(:handles) AS handle, unnest(:pinned_track_ids) AS pinned_track_id) AS data_table
+        WHERE users.is_current = True AND users.updated_at < '2023-01-06 08:35:00' AND users.handle = data_table.handle;"""
```

Comment:
is it ok to use handle_lc since that's indexed? querying against handle is very slow

Reply:
nice, added in https://github.com/AudiusProject/audius-protocol/pull/4564/commits/98a20525a3650036049df28c3296e948f8c3ee47